src/matcha_ml/state/remote_state_manager.py

```diff
@@ -191,8 +191,11 @@ def provision_remote_state(
 
         account_name, container_name, resource_group_name = template_runner.provision()
         self._write_matcha_config(account_name, container_name, resource_group_name)
-
-        print_status(build_step_success_status("Provisioning is complete!"))
+        print_status(
+            build_step_success_status(
+                "Provisioning of the remote state storage is complete!"
```

Comment:
should it be `Matcha resources` instead of `remote state storage` as changed in `deprovision_remote_state`

Reply:
Great spot, updated.